app/helpers/date_time_helper.rb

```diff
@@ -44,4 +45,14 @@ def api_date_format(date_time)
       end
     end
   end
+
+  def to_be_debated_on_in_words(date, today = Date.current)
+    return unless date.present?
```

Comment:
You don't need this check here now since we're checking in the view for the presence of the date